packages/report-server/src/reportBuilder/transform/parser/TransformParser.ts

```diff
@@ -14,11 +15,11 @@ type RowLookup = {
  * Lookups object for rows within the transform data
  *
  * eg. if rows = [{BCD1: 5}, {BCD1: 8}, {BCD1: 4}], and currentRow = 1
- * '$row.BCD1' => 8
- * '$all.BCD1' => [5, 8, 4]
- * '$allPrevious.BCD1' => [5, 8]
- * '$where(f($otherRow) = $otherRow.BCD1 < $row.BCD1).BCD1' => [5, 4]
- * '$row.BCD1 + sum($allPrevious.BCD1)' => 21
+ * '_row.BCD1' => 8
+ * '_all.BCD1' => [5, 8, 4]
+ * '_allPrevious.BCD1' => [5, 8]
+ * '_where(f(_otherRow) = _otherRow.BCD1 < _row.BCD1).BCD1' => [5, 4]
+ * '_row.BCD1 + sum(_allPrevious.BCD1)' => 21
```

Comment:
`#` is a comment in mathjs, so we can't use that to prefix the lookups unfortunately 😢  I tried `@` as well, but that gives a syntax error, so I went with `_` in the end... Would prefer something a bit more visually obvious any suggestions?

Reply:
Ignore this, got `@` working in the end! mathjs is so configurable it's amazing! 🎉